### ISSUE TITLE
unnecessary url in state?

### ISSUE THREAD

**Issue by cdrani** [JasonFritsche/BreweryFinder#23]
`url` does not need to be stored in state. There's no apparent reason for it to be there.
```js
  async fetchBreweryData() {
    try {
      this.setState({ isLoaded: false });
      const { searchTerm, searchByParam } = this.state;
      const url = `https://api.openbrewerydb.org/breweries?by_${searchByParam}=${searchTerm}`;
      const data = await fetch(url);
      const jsonData = await data.json();
      const cleanData = this.filterResults(jsonData);
      this.setState({
        isLoaded: true,
        items: cleanData
      });
    } catch (error) {
      console.log(error);
    }
  }
```

Additionally, `isLoaded` could also be replaced with [React.Suspense](https://reactjs.org/docs/code-splitting.html#suspense), although there's no real benefit from this change other than not needing to worry about another state value.

**Comment by cdrani** [comment]
// @JasonFritsche: What do you think about this? Should I proceed?

**Comment by JasonFritsche** [comment]
Have you tested out the the app after removing url from the state?

**Comment by cdrani** [comment]
Yes. PR coming soon.

**Comment by JasonFritsche** [comment]
@cdrani I've merged PR #38 , thanks for your contribution.
